Mathlib/Mathport/Syntax.lean

```diff
@@ -580,7 +581,18 @@ syntax (name := aliasLR) "alias " ident " ↔ " (".." <|> (binderIdent binderIde
 syntax (name := explode) "#explode " ident : command
 
 syntax (name := open_locale) "open_locale" (ppSpace ident)* : command
+macro_rules | `(open_locale $ids:ident*) => `(command| open $[$ids:ident]*)
+
 syntax (name := localized) "localized " "[" ident "] " command : command
+-- The implementation completely ignores the namespace argument, let's just
+-- hope that localized is only used the corresponding namespace. :shrug:
```

Comment:
In mathport, we know what namespace we are in. Would it be better to output `scoped` instead of `localized` when the two coincide?

Reply:
I think that would be the best solution, we should translate localized notation to scoped notation.  I'm going to merge this one for now and as a fallback.

Reply:
A more faithful translation would change the namespace to the ident and declare the scoped notation there (using [`withWeakNamespace`](https://github.com/leanprover-community/mathport/blob/master/Mathport/Util/Misc.lean#L159-L165) to set the namespace without causing local things to immediately go out of scope)

Reply:
That's a fancy combinator!  Let's steal it: https://github.com/leanprover-community/mathlib4/pull/151